src/components/studies/app-design/GeneralContactAndSupportSection.tsx

```diff
@@ -112,9 +106,6 @@ const GeneralContactAndSupportSection: React.FunctionComponent<GeneralContactAnd
                 contactLeadInfo: newContactLeadObject,
               })
             }}
-            onBlur={() =>
-              updateAppDesignInfo(AppDesignUpdateTypes.UPDATE_STUDY_CONTACTS)
-            }
             multiline
```

Comment:
please remove unnecessary 
 `multiline rows={1} rowsMax={1}`  and make those props optional on the text input wrapper